runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -280,6 +281,13 @@
          * The number of times to retry invalidate when a layout change is expected.
          */
         @Default int invalidateRetry = 5;
+
+        /**
+         * Represents filtering logic to be applied on the inbound messages received by Netty Client
+         * Router. If filters are not null, Netty Client Router add a filter handler and configures it
+         * with provided filters. If filters are null, no filter handler will be added to Netty's pipeline.
+         */
+        @Default List<MsgHandlingFilter> nettyClientInboundMsgFilters = null;
```

Comment:
All runtimes in test scenarios (including the clients on the servers) should now default this to non-null to enable message filtering. We should see if there is an easier way to enable this for all test cases.

Reply:
There won’t be any need to make any changes for current tests. If they want to add filtering behavior then they have to add inboundFilters.